### ISSUE TITLE
/v with modelID

### ISSUE THREAD

**Issue by wtfrazr** [Pottus/Texture-Studio#75]
/v doesn't work with SA:MP vehicle modelID. Only names can be put into the command and that doesn't always work for some specific vehicles.

**Comment by Crayder** [comment]
Again, /v has nothing to do with TS. That's an fsdebug command. Use /tcar.

**Comment by wtfrazr** [comment]
Yeah, but that is temporary. I want a vehicle that works like /v. /avcar or whatever it is bugged.
